src/main/kotlin/org/jitsi/nlj/dtls/TlsClientImpl.kt

```diff
@@ -30,24 +30,36 @@ import org.bouncycastle.crypto.tls.TlsCredentials
 import org.bouncycastle.crypto.tls.TlsSRTPUtils
 import org.bouncycastle.crypto.tls.TlsUtils
 import org.bouncycastle.crypto.tls.UseSRTPData
+import org.jitsi.nlj.util.getLogger
 import java.util.*
 
 /**
  * Implementation of [DefaultTlsClient].
  */
-class TlsClientImpl : DefaultTlsClient() {
+class TlsClientImpl(
+        /**
+         * The function to call when the server certificate is available.
+         */
+        private val notifyServerCertificate: (Certificate) -> Unit)
+        : DefaultTlsClient() {
```

Comment:
the spacing i've done for class params is to have it match the member variables, since they basically are members, and closing the class arg list on a new line, so:

```
class TlsClientImpl(
    /**
     * The function to call when the server certificate is available.
     */
    private val notifyServerCertificate: (Certificate) -> Unit)
) : DefaultTlsClient() {
    private val logger = getLogger(this.javaClass)
    ...
}